### PR TITLE
chore: add Atharv-777

### PR DESCRIPTION
**Pull request by Atharv-777**
## Changes proposed

Added Atharv-777.json file

## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/1388"></a>

### CODE REVIEW

**Review by Panquesito7 — CHANGES_REQUESTED** [review]
Hey there! Please revert the changes in the `package-lock.json` and `package.json` files. Let us know if you need any help. Thanks. :)

**Review by Pradumnasaraf — APPROVED** [review]
Thank you for the changes. Looks good to merge 

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :ok_hand: This code looks good&nbsp; | &nbsp;**Tags:** Reusable

File: public/data/Atharv-777.json [hunk 1]
```diff
@@ -0,0 +1,24 @@
+
```
[the remark this answers is not in the document]
Comment (Pradumnasaraf) [reply public/data/Atharv-777.json:1]: Hey! David. Don't the action the will fix that?

**Review by emmalearnscode — APPROVED** [review]
__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :ok_hand: This code looks good

**Review by Pradumnasaraf — APPROVED** [review]
Looks good to merge

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :ok_hand: This code looks good&nbsp; | &nbsp;**Tags:** Maintainable

**Review by Panquesito7 — APPROVED** [review]
LGTM 🚀 Thank you for your contribution! 😄👍


__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :ok_hand: This code looks good&nbsp; | &nbsp;**Tags:** Elegant

**Pull request merged**